index.html

```diff
@@ -83,26 +112,64 @@
                     <!-- Stop ignoring -->
                     {% for student in site.data.students %}
                     <div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
-                        <div class="card stu-card">
-                            <div class="avatar">
-                                {% if student.image %}
-                                <img src="images/students/{{student.image}}"
-                                alt="{{student.name}}"
-                                width="200px" height="200px">
-                                {% else %}
-                                <img src="images/mentors/member.png"
-                                alt="{{student.name}}"
-                                width="200px" height="200px">
-                                {% endif %}
-                                <p style="padding-top:3px;width: 200px;display: inline-block;margin-top: 9px;">
-                                    <i class="fa fa-code"></i>
-                                    &nbsp;
-                                    <span class="stu-name card-text" title="{{student.name}}">
-                                        {{student.name}}
-                                    </span>
-                                </p>
+                        <div class="card-wrapper">
+                            <div class="flip-container" ontouchstart="this.classList.toggle('hover');">
+                                <div class="flipper">
+                                    <div class="front">
+                                        <!-- front content -->
+                                        <div class="card stu-card">
```

Comment:
Can you please move 
```
 <div class="flip-container" ontouchstart="this.classList.toggle('hover');">
                                <div class="flipper">
                                    <div class="front">
```
so that card flips only on going to the `avatar` class which is the actual content